frontend/javascripts/admin/task/task_list_view.tsx

```diff
@@ -237,6 +236,176 @@ function TaskListView({ initialFieldValues }: Props) {
+      width: 200,
+      sorter: Utils.localeCompareBy<APITask>((task) => task.type.summary),
+      render: (taskType: APITaskType) => (
+        <a href={`/taskTypes#${taskType.id}`}>{taskType.summary}</a>
+      ),
+    },
+    {
+      title: "Dataset",
+      dataIndex: "dataSet",
+      key: "dataSet",
+      sorter: Utils.localeCompareBy<APITask>((task) => task.dataSet),
+    },
+    {
+      title: "Stats",
+      dataIndex: "status",
+      key: "status",
+      render: (status: APITask["status"], task: APITask) => (
+        <div className="nowrap">
+          <span title="Pending Instances">
+            <PlayCircleOutlined className="icon-margin-right" />
+            {status.pending}
+          </span>
+          <br />
+          <span title="Active Instances">
+            <ForkOutlined className="icon-margin-right" />
+            {status.active}
+          </span>
+          <br />
+          <span title="Finished Instances">
+            <CheckCircleOutlined className="icon-margin-right" />
+            {status.finished}
+          </span>
+          <br />
+          <span title="Annotation Time">
+            <ClockCircleOutlined className="icon-margin-right" />
+            {formatSeconds((task.tracingTime || 0) / 1000)}
+          </span>
+        </div>
+      ),
+      filters: [
+        {
+          text: "Has Pending Instances",
+          value: "pending",
+        },
+        {
+          text: "Has Active Instances",
+          value: "active",
+        },
+        {
+          text: "Has Finished Instances",
+          value: "finished",
+        },
+      ],
+      onFilter: (key: any, task: APITask) => task.status[key as unknown as keyof TaskStatus] > 0,
+    },
+    {
+      title: "Edit Position / Bounding Box",
+      dataIndex: "editPosition",
+      key: "editPosition",
+      width: 150,
+      render: (_, task: APITask) => (
+        <div className="nowrap">
+          {formatTuple(task.editPosition)} <br />
+          <span>{formatTuple(task.boundingBoxVec6)}</span>
+        </div>
+      ),
+    },
+    {
+      title: "Experience",
+      dataIndex: "neededExperience",
+      key: "neededExperience",
+      sorter: Utils.localeCompareBy<APITask>((task) => task.neededExperience.domain),
+      width: 250,
+      render: (neededExperience: APITask["neededExperience"]) =>
+        neededExperience.domain !== "" || neededExperience.value > 0 ? (
+          <Tag>
+            {neededExperience.domain} : {neededExperience.value}
+          </Tag>
+        ) : null,
+    },
+    {
+      title: "Creation Date",
+      dataIndex: "created",
+      key: "created",
+      width: 200,
+      sorter: Utils.compareBy<APITask>((task) => task.created),
+      render: (created: APITask["created"]) => <FormattedDate timestamp={created} />,
+      defaultSortOrder: "descend",
+    },
+    {
+      title: "Action",
+      key: "actions",
+      width: 170,
+      fixed: "right",
+      render: (_, task: APITask) => (
```

Comment:
Here you are shadowing the lodash variable. That's why calls within this rendering function using lodash do not work. See my highlighted example below. Just give this a different name and it should work:
```suggestion
      render: (_unused, task: APITask) => (
```